### DISCUSSION TITLE
New stuff to work on...?

### DISCUSSION THREAD

**Discussion by lirantal** [Pull-Request-Community/pull-request-community-website#230] [Ideas]
Hi folks, what should we work on to improve?
Some ideas like...
- Light mode or dark mode toggle?
- Tests - tests are always good and I haven't seen any in the repo

**Comment by MichalPorag** [comment]
Hi @lirantal tests are a great idea! I'm opening an issue for that.

**Comment by InbarDanieli** [comment]
Hi!
What if we add a "scroll to top" button that will appear after you scroll some distance on the page?

**Reply by lirantal** [reply]
@InbarDanieli I like that!
